src/app/constructor/map/map-constructor.component.ts

```diff
@@ -155,6 +156,12 @@ export class MapConstructorComponent implements OnInit {
         }
     }
 
+    private setInitCoordinates(sensor: Device) {
+        const offset = this.edittedDevices.length * 100;
```

Comment:
From name of the function we can understand that it's offset of sensor which will be placed on map view picture. The offset calculates relatively top-left angle of map-view picture.